server/src/main/java/io/druid/server/coordinator/helper/DruidCoordinatorBalancer.java

```diff
@@ -191,6 +198,7 @@ public void execute()
             segmentToMove.getIdentifier(),
             callback
         );
+        return true;
       }
       catch (Exception e) {
```

Comment:
Shouldn't segment be removed from `currentlyMovingSegments` in the end of this catch block?